src/wp-includes/class-wp-theme-json.php

```diff
@@ -2764,6 +2807,21 @@ protected static function remove_insecure_styles( $input ) {
 				}
 			}
 		}
+
+		// Ensure indirect properties not handled by `compute_style_properties` are allowed.
+		foreach ( static::INDIRECT_PROPERTIES_METADATA as $property => $paths ) {
+			foreach ( $paths as $path ) {
+				$value = _wp_array_get( $input, $path, array() );
+				if (
+					null !== $value &&
+					! is_array( $value ) &&
+					static::is_safe_css_declaration( $property, $value )
+				) {
+					_wp_array_set( $output, $path, $value );
+				}
+			}
+		}
```

Comment:
This code pattern is a duplicate from `remove_insecure_settings()` method. Instead of duplicating the code, I suggest encapsulating it into a reusable private method. Then invoke the method in both places.

I'll push a change shortly.

Reply:
Resolved in https://github.com/WordPress/wordpress-develop/pull/3742/commits/c05ca999b5875bb06eda819ce723d3adb1b42307.